_data/simple-icons.json

```
@@ -2450,6 +2450,11 @@
             "hex": "003791",
             "source": "https://commons.wikimedia.org/wiki/File:PlayStation_4_logo_and_wordmark.svg"
         },
+        {
+            "title": "Plesk",
+            "hex": "52BBE6",
+            "source": "https://scdn1.plesk.com/wp-content/uploads/2017/07/24160129/plesk_logo.zip"
```

Can you update this to just the Plesk Brand Guidelines page, that link is more informative in general and will probably last longer (if they update the zip it seems the URL would be outdated).

```suggestion
            "source": "https://www.plesk.com/brand/"
```